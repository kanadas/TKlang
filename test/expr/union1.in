(\x -> if x == (1 @ 1) then 1 @ true else 2 @ false) (1 @ 5)
