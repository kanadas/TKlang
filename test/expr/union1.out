expr/union1.in
Not concrete type: (int+#4)
